ote/src/cljs/ote/app/controller/transport_service.cljs

```diff
@@ -284,8 +284,9 @@
   DeleteTransportServiceResponse
   (process-event [{response :response} app]
     (let [filtered-map (filter #(not= (:ote.db.transport-service/id %) (int response)) (get app :transport-service-vector))]
```

Comment:
do we still need to filter this collection if we are fetching the data again?

Reply:
Nope, we don't need.
But delete isn't working correctly, so I need to ensure functionality first.